LEAF_Nexus/css/employeeSelector.css

```diff
@@ -27,6 +27,15 @@
     cursor: pointer;
 }
 
+.employeeSelectorDisable{
+    width: 99%;
+    font-size: 14px;
+    background-color: #ff7777 !important;
+    border: 1px solid black;
+    padding: 2px;
+    cursor: pointer;
+}
+
```

Comment:
Had a thought about this yesterday night; may be worth it to revisit and make this grey. At least something to bring up to MGao/Su when demoed.

Reply:
Yeah, of course this is something that can be changed real quick once we demo. Thank you for the feed back!